lib/paper_trail/attribute_serializers/attribute_serializer.rb

```diff
@@ -0,0 +1,22 @@
+require "paper_trail/attribute_serializers/array_serializer"
+
+module PaperTrail
+  module AttributeSerializers
+    # Values returned by some Active Record serializers are
+    # not suited for JSON encoding. This factory
+    # replaces certain default Active Record serializers
+    # with custom PaperTrail ones.
```

Comment:
A good, helpful comment. I'd change "JSON encoding" to "writing JSON to a text column.".

This phrasing disabiguates JSON-in-text-column from native postgres json(b).